### PR TITLE
Linalg readme

### PR DESCRIPTION
**Pull request by OXPHOS**

### CODE REVIEW

File: doc/readme/README_linalg.md [hunk 1]
```diff
@@ -0,0 +1,190 @@
+## Internal linear algebra library
```
Comment (karlnapf) [inline doc/readme/README_linalg.md]: Did you write this guy from scratch?

Comment (OXPHOS) [inline doc/readme/README_linalg.md]: It was modified from the old [README](https://github.com/shogun-toolbox/shogun/wiki/README_linalg) but I was not sure where it was located and I don't want to edit it directly

**Comment by karlnapf** [comment]
I think I am in favour of just deleting any readmes on linalg. The code speaks for itself and is not too messed up (mostly at least ;) )
So just maybe remove file and in wiki?

**Comment by vigsterkr** [comment]
++ for wiki!

**Pull request closed without merging**